### ISSUE TITLE
Feature request: Add option to disable smoothing

### ISSUE THREAD

**Issue by andrewdbate** [stsdc/monitor#298]
Please add an option to disable smoothing. This is something which is supported by Gnome System Monitor.

Here is an example CPU plot from Gnome System Monitor (of a stacked chart):
![smoothing-enabled](https://user-images.githubusercontent.com/2998569/150430399-17830832-10d0-46ad-9170-13c3d501b209.png)
Here is how the plot can look with **smoothing disabled**:
![smoothing-disabled](https://user-images.githubusercontent.com/2998569/150430450-f819fe79-129e-4c2e-8164-36f15d86043f.png)

The option to disable smooth plots is within the Gnome System Monitor preferences dialog:
![gnome-system-monitor-preferences](https://user-images.githubusercontent.com/2998569/150430679-732ed49a-251f-4d48-ac66-ab8b55ee889a.png)

Here is another plot from Gnome System Monitor (not a stacked chart):
![with-smoothing-not-stacked](https://user-images.githubusercontent.com/2998569/150431282-950d29d4-3717-46bf-82aa-e2cfe1ad86fd.png)
And the same kind of plot with **smoothing disabled**:
![without-smoothing-not-stacked](https://user-images.githubusercontent.com/2998569/150431321-ad99a0ee-1a8a-4bfb-a8f9-b063bd7dd9d3.png)

The rationale for this feature request: I find the CPU plots easier to read with the smooth lines disabled because having slightly less detail makes it easier to identify what is going on with only a quick glance.

**Comment by andrewdbate** [comment]
Here is a video of how Gnome System Monitor handles the turning on/off of smooth lines (43 seconds):

https://user-images.githubusercontent.com/2998569/159173512-1819b0bf-7a5e-49e1-8eb9-4a5670eb0bfa.mp4

As you can see, in Gnome System Monitor, when you toggle on or off "Draw CPU chart as smooth graph", the chart is cleared and it redraws from the beginning.

It would be slightly more elegant to draw either with or without smooth lines starting from the next clock tick (and that way the user can more easily see the difference between smooth and not smooth) instead of clearing the graph and starting again.

**Comment by andrewdbate** [comment]
Thinking again about the wording on which I commented above: It would be even more concise to say "Draw smooth lines on CPU chart (requires restart)".

This would take up less horizontal width.

**Comment by stsdc** [comment]
Yes, I'll try to make it restartless. However I'll wait until a new version of live-chart will be released. Hope it will natively support stacked charts.
